modules/siddhi-core/src/main/java/org/wso2/siddhi/core/executor/function/InstanceOfStringFunctionExecutor.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.siddhi.core.executor.function;
+
+import org.wso2.siddhi.core.config.ExecutionPlanContext;
+import org.wso2.siddhi.core.exception.ExecutionPlanRuntimeException;
+import org.wso2.siddhi.core.executor.ExpressionExecutor;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.query.api.exception.ExecutionPlanValidationException;
+
+/**
+ * instanceOfString(input)
+ * This method returns true if and only if the input is a instance of String
+ * input - the value to check for String instance eg: WSO2
+ * Accept Type(s) for instanceOfString(input);
+ *         inputSequence : BOOLEAN, STRING, INT, FLOAT, DOUBLE, LONG
```

Comment:
Thanks @dilini-muthumala, Changes are done in pull request - https://github.com/wso2/siddhi/pull/72